bundle/bundle.go

```diff
@@ -14,6 +14,15 @@ import (
 	pkgErrors "github.com/pkg/errors"
 )
 
+// CNABSpecVersion represents the CNAB Spec version of the Bundle
+// Schema injected at build-time.
```

Comment:
So if I use cnab-go in my project, I need to add build flags to get the right version?
Why don't we provide a default value, as this code base is targeting a specific version ?

Reply:
Hmm, good point.  In the latest changeset, I've added defaults.

I may explore updating the Makefile logic around fetching schema at build-time to run via a go program such that the canonical default versions are only in one place.

Reply:
Ok, added the go program... now the schema versions are set in one place (library code) and the program ingests these to determine which remote schemas to pull at build-time.